crates/services/producer/src/block_producer.rs

```diff
@@ -88,14 +88,14 @@ where
         max_gas: Word,
     ) -> anyhow::Result<UncommittedResult<StorageTransaction<ExecutorDB>>> {
         //  - get previous block info (hash, root, etc)
-        //  - select best da_height from relayer
-        //  - get available txs from txpool
+        //  - select best `da_height` from Relayer
+        //  - get available txs from TxPool
         //  - select best txs based on factors like:
         //      1. fees
         //      2. parallel throughput
-        //  - Execute block with production mode to correctly malleate txs outputs and block headers
+        //  - Execute block with production mode to correctly malleable txs outputs and block headers
```

Comment:
Here's another "malleable" being used as a verb.

Reply:
Maybe "modify" instead?

Reply:
We could go with modify, but malleate carries some context from the bitcoin world which is useful conceptually.